simulationTool/components/HelpSection/TutorialPanel.vue

```diff
@@ -0,0 +1,145 @@
+<script>
+import SectionHeader from '../SectionHeader.vue';
+import TutorialCreateEnsemble from './TutorialCreateEnsemble.vue';
+import TutorialCreateScenario from './TutorialCreateScenario.vue';
+import TutorialEnsembleDetails from './TutorialEnsembleDetails.vue';
+import TutorialEnsembleOverview from './TutorialEnsembleOverview.vue';
+import TutorialLandingPage from './TutorialLandingPage.vue';
+import TutorialModelInformation from './TutorialModelInformation.vue';
+import TutorialModelOverview from './TutorialModelOverview.vue';
+import TutorialScenarioOverview from './TutorialScenarioOverview.vue';
+import TutorialSzenarioDetails from './TutorialSzenarioDetails.vue';
+
+export default {
+    name: "TutorialPanel",
+    components: {
+        SectionHeader,
+        TutorialLandingPage,
+        TutorialModelOverview,
+        TutorialModelInformation,
+        TutorialScenarioOverview,
+        TutorialCreateScenario,
+        TutorialSzenarioDetails,
+        TutorialEnsembleOverview,
+        TutorialEnsembleDetails,
+        TutorialCreateEnsemble
+    },
+    data() {
+        return {
+            currentIndex: 0,
+            views: [
+                { name: 'TutorialLandingPage' },
+                { name: 'TutorialModelOverview' },
+                { name: 'TutorialModelInformation' },
+                { name: 'TutorialScenarioOverview' },
+                { name: 'TutorialCreateScenario' },
+                { name: 'TutorialSzenarioDetails' },
+                { name: 'TutorialEnsembleOverview' },
+                { name: 'TutorialEnsembleDetails' },
+                { name: 'TutorialCreateEnsemble' }
+            ]
+        };
+    },
+    methods: {
+        nextView() {
+            if (this.currentIndex < this.views.length - 1) {
+                this.currentIndex++;
+            }
+        },
+        previousView() {
+            if (this.currentIndex > 0) {
+                this.currentIndex--;
+            }
+        }
+    }
+};
+</script>
+
+<template>
+    <div class="tutorial-panel">
+        <SectionHeader :title="$t('additional:modules.tools.simulationTool.help')" icon="bi bi-question-circle-fill" />
+        <div class="segment-wrapper">
+            <div class="element-wrapper">
+                <component :is="views[currentIndex].name" />
+            </div>
+            <div class="navigation-buttons btn-light">
+                <button @click="previousView" :disabled="currentIndex === 0" class="arrow-button">
+                    <i class="bi bi-chevron-left"></i>
+                </button>
+                <button @click="nextView" :disabled="currentIndex === views.length - 1" class="arrow-button">
+                    <i class="bi bi-chevron-right"></i>
+                </button>
+            </div>
+        </div>
+    </div>
+</template>
+
+<style lang="scss" scoped>
```

Comment:
A different approach to the before mentioned `TutorialPage` component would be to remove the `scoped` from this `<style>` so the style defined here would apply to all other components. You could then remove the `<style>` from the `Tutorial*.vue` files. IMHO this is not the cleanest way but it should work, too.